tests/unit/core/components/TestInstallButton.js

```diff
@@ -37,7 +38,7 @@ describe('<InstallButton />', () => {
   });
 
   const render = (props) =>
-    shallowRender(<InstallButtonBase {...renderProps(props)} />);
+    shallow(<InstallButtonBase {...renderProps(props)} />);
 
   const renderToDom = (props) => findDOMNode(
```

Comment:
oh yeah probably but I tried to avoid moving anything that wasn't `shallowRender`. I didn't want the diff to be too large and figured we can do another PR for `findDOMNode`.